src/main/java/lotto/domain/vo/WinningLotto.java

```diff
@@ -0,0 +1,55 @@
+package lotto.domain.vo;
+
+import java.util.List;
+import lotto.exception.DomainException;
+import lotto.exception.ExceptionCode;
+
+public class WinningLotto {
+    private final Lotto lotto;
+    private final BonusNumber bonusNumber;
+
+    private WinningLotto(Lotto lotto, BonusNumber bonusNumber) {
+        validateCombinations(lotto, bonusNumber);
+        this.lotto = lotto;
+        this.bonusNumber = bonusNumber;
+    }
+
+    public static WinningLotto of(List<Integer> numbers, Integer bonusNumber) {
+        return new WinningLotto(
+                Lotto.of(numbers),
+                BonusNumber.from(bonusNumber)
+        );
+    }
+
+    private void validateCombinations(Lotto lotto, BonusNumber bonusNumber) {
+        Integer targetNumber = bonusNumber.getBonusNumber();
+        if (!lotto.isIncluding(targetNumber)) {
```

Comment:
[마이너] 사소하지만, `!lotto.isIncluding()`은 생각을 두번 하게 되는 메서드 형식인 것 같습니다. `lotto.isExcluding()`처럼 한번에 의미를 파악하는 형식도 추천 드립니다

Reply:
음.. 기존에 있던 isIncluding 이라는 메서드를 재활용 하느라 이렇게 사용했던것 같은데
여기서만 그렇게 사용될거라면 

```suggestion

    private void validateCombinations(Lotto lotto, BonusNumber bonusNumber) {
        Integer targetNumber = bonusNumber.getBonusNumber();
        boolean isExcluding = !lotto.isIncluding(targetNumber);
        if (isExcluding) {
            return;
        }
        throw new DomainException(ExceptionCode.BONUS_NUMBER_DUPLICATED);
    }

```
이렇게 해서 의미를 제대로 부여하도록 하겠습니다!